README.md

```diff
@@ -83,7 +86,7 @@ It will produce `{bin-name}-heap.json` file in the root of the workdir, that can
 
 ## Tests & Benchmarks
 
-- To run all current tests use:
+- To run basic unit tests use:

```

Comment:
Formula "basic unit" tests is misleading and reductive. Most integration tests are included. In fact only RPC integration tests are missing.

Perhaps replace with "unit and most integration" tests

Reply:
fixed